docs/src/user/variables-guide.rst

```diff
@@ -187,6 +196,26 @@ Ansible module. In OS Migrate they are named as follows::
     os_migrate_conversion_rhsm_syspurpose
     os_migrate_conversion_rhsm_username
 
+Enabling password-based SSH access to the conversion hosts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Ill rebase after #452 is merged, so the text in the prev lines is removed from this PR.

Reply:
done